### PR TITLE
New formula i3status

### PR DESCRIPTION
**Pull request by afh**

### CODE REVIEW

File: i3.rb [hunk 1]
```diff
@@ -23,6 +22,6 @@ def install
   end
 
   test do
-    system "#{bin}/i3", "-v"
+    assert_match /^i3 version [-0-9a-z.]+.*2009-\d{4}.*$/, `#{bin}/i3 -v`
```
Comment (DomT4) [inline i3.rb]: You should just be able to `assert_match /#{version}/, shell_output("#{bin}/i3 -v")` here.

File: i3status.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class I3status < Formula
+  homepage "http://i3wm.org/i3status/"
+  url "http://i3wm.org/i3status/i3status-2.8.tar.bz2"
+  sha1 "34c874c23fef0c5652f4449c6c8e4f8df57bfeb9"
```
Comment (DomT4) [inline i3status.rb]: `sha256` please

File: i3status.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+class I3status < Formula
+  homepage "http://i3wm.org/i3status/"
```
Comment (DomT4) [inline i3status.rb]: Add a `desc` above the homepage please. 80 Characters or less.

File: i3status.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  depends_on "yajl"
+  depends_on :x11
+
+  def install
+    system "make"
+    system "make", "PREFIX=#{prefix}", "install"
+  end
+
+  test do
+    assert_match /^i3status [-0-9a-z.]+.*2008-\d{4}.*$/, `#{bin}/i3status -v`
```
Comment (DomT4) [inline i3status.rb]: You should just be able to `assert_match /#{version}/, shell_output("#{bin}/i3status -v")` here.

**Comment by DomT4** [comment]
Any interest in this still?

**Comment by afh** [comment]
Yes, but since the initial pull request the project has seen some changes and I'm hoping to get some minor [compatibility edits](https://github.com/i3/i3status/pull/40) into the upstream repo.

**Comment by DomT4** [comment]
Alright. No worries, as long as we know the PR is active it won't get killed.

**Comment by afh** [comment]
Unfortunately `brew test i3status` fails due to an [issue](https://github.com/i3/i3status/issues/41) in the upstream repository.

**Comment by DomT4** [comment]
If it needs to use `git` you can get away with doing:

```ruby
url "https://github.com/i3/i3status.git",
    :tag => "2.9",
    :revision => "523f7b269b7f75d6075ad38b09d26a2b9697ca88"
```

Might work.

**Comment by afh** [comment]
There is another source for the official i3status release archives. I've updated the formula and this pull-request accordingly and it is know ready for review (and hopefully also approval ;)

**Comment by afh** [comment]
@DomT4 I've updated this formula and deem it ready for review and inclusion into `homebrew-x11` if it meets your demands.

**Comment by dunn** [comment]
It's everyone's favorite test failure:
```
==> /usr/local/Cellar/i3status/2.9/bin/i3status -v
Error: homebrew/x11/i3status: failed
invalid byte sequence in US-ASCII
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/minitest/unit.rb:302:in `=~'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/minitest/unit.rb:302:in `assert_match'
/usr/local/Library/Taps/homebrew/homebrew-x11/i3status.rb:55:in `block in <class:I3status>'
/usr/local/Library/Homebrew/formula.rb:1277:in `block in run_test'
/usr/local/Library/Homebrew/extend/fileutils.rb:17:in `mktemp'
/usr/local/Library/Homebrew/formula.rb:1273:in `run_test'
/usr/local/Library/Homebrew/test.rb:28:in `block in <main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/timeout.rb:66:in `timeout'
/usr/local/Library/Homebrew/test.rb:27:in `<main>'
```
Could you try adding `ENV["LC_ALL"] = "en_US.UTF-8"` to the beginning of the test block?

**Comment by afh** [comment]
@dunn Unfortunately adding `ENV["LC_ALL"] = "en_US.UTF-8"` did not remedy everyone's favourite test failure.

Luckily another approached worked just fine, please have a look.

File: i3status.rb [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+  depends_on :x11
+  depends_on "yajl"
+  depends_on "confuse"
+  depends_on "pulseaudio"
+  depends_on "asciidoc" => :build
+  depends_on "i3" => :recommended
+
+  def install
+    ENV.append_to_cflags "-I#{HOMEBREW_PREFIX}/include"
```
Comment (DomT4) [inline i3status.rb]: This and the line below should be happening automatically. Do you see problems if you remove them?

Comment (afh) [inline i3status.rb]: @DomT4 No building the formula works fine without them. I removed those lines and updated with PR accordingly.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-x11/commit/ff700fea8a15429bf928af4acb4eab2e87a95a00. Thank you once more @afh! :smiley_cat:

**Comment by afh** [comment]
Thanks for merging!

**Pull request closed without merging**